lesson_20/homework_20.py

```diff
@@ -0,0 +1,119 @@
+"""Homework 20."""
+import csv
+import logging
+import sqlite3
+
+"""
+Створіть базу даних для інтернет-магазину з наступними таблицями:
+products: таблиця для зберігання інформації про продукти,
+включаючи назву, опис, ціну тощо.
+categories: таблиця для категорій продуктів.
+products повинна мати зовнішній ключ на таблицю categories.
+Напишіть SQL-скрипт для створення зазначених таблиць.
+Внесіть декілька рядків даних в кожну таблицю
+Виконайте JOIN-запит, який повертає інформацію про продукти
+та назву їх категорій
+"""
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+def create_tables():
+    """Create tables."""
+    with sqlite3.connect('internet_shop.db') as conn:
+        cursor = conn.cursor()
```

Comment:
One possible option here organize class - during init create connection and during magic method del close this connection.  In your case as minimum DB name should be placed as constant to the top of module